README.md

```diff
@@ -50,6 +50,8 @@ $ yarn build
 $ yarn service:timeseriesWithoutAggregateMigration
 ```
 
+ℹ️ Running `yarn build` is a prerequisite as it generates scripts used by `yarn service:timeseriesWithoutAggregateMigration`. Running `yarn start` after a `yarn build` would delete those scripts.
```

Comment:
`yarn start` can never be used to build services, one must use `yarn watch` instead: https://github.com/cozy/cozy.github.io/blob/dev/src/howTos/dev/services.md#build
So I would rephrase like this: 
```
ℹ️ Services can only be built through `yarn watch` or `yarn build`, but never `yarn start`, which actually deletes the services files. See the [services documentation](https://docs.cozy.io/en/howTos/dev/services/#build) for more insights.
```

Reply:
What is this... is this a subject for a "PdC"? 😅 

(i'm not used to work with services)

I'll do the edit tomorrow

Reply:
![image](https://user-images.githubusercontent.com/7010087/188934432-5e1e851c-186b-472a-8fd5-09e59938e9ca.png)